test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -443,8 +442,10 @@ def doc_schema(module_name, for_collection=False, deprecated_module=False):
     if for_collection:
         # Optional
         doc_schema_dict['version_added'] = version(for_collection=True)
+        doc_schema_dict['author'] = Any(None, list_string_types, *string_types)
```

Comment:
@mattclay what about doing this?

Reply:
This looks good to me. I'd like to know what @acozine and @gundalow think about the change.

Also, this change requires a changelog fragment.